### PR TITLE
DOC-2411: List with start attribute separated by  tags resulted in wrong numbering of the lists.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2411

Site: [Staging branch](http://docs-feature-711-doc-2411tiny-10585.staging.tiny.cloud/docs/tinymce/latest/7.1.1-release-notes/#powerpaste)

Changes:
* add fix for TINY-10585 to the release notes.

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Review by hamza0867 — APPROVED** [review]

File: modules/ROOT/pages/7.1.1-release-notes.adoc [hunk 1]
```diff
@@ -59,6 +59,30 @@ For information on the **<Open source plugin name>** plugin, see xref:<plugincod
...
+=== PowerPaste
+
+The {productname} {release-version} release includes an accompanying release of the **PowerPaste** premium plugin.
+
+**PowerPaste** includes the following fix.
+
+==== List with start attribute separated by `p` tags resulted in wrong numbering of the lists.
+// TINY-10585
+
+Previously, when content from Microsoft Word contained a list that starts with a number grater than 1, and is pasted into the editor with `powerpaste` enabled, it caused all list items to display the same number.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/7.1.1-release-notes.adoc:71]:
```suggestion
Previously, when content from Microsoft Word contained a list that starts with a number greater than 1, and is pasted into the editor with `powerpaste` enabled, it caused all list items to display the same number.
```

File: modules/ROOT/pages/7.1.1-release-notes.adoc [hunk 2]
```diff
@@ -59,6 +59,30 @@ For information on the **<Open source plugin name>** plugin, see xref:<plugincod
 
 The following premium plugin updates were released alongside {productname} {release-version}.
 
+=== PowerPaste
+
+The {productname} {release-version} release includes an accompanying release of the **PowerPaste** premium plugin.
+
+**PowerPaste** includes the following fix.
+
+==== List with start attribute separated by `p` tags resulted in wrong numbering of the lists.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/7.1.1-release-notes.adoc:68]: Please improve this heading

**Review by tiny-ben-tran — APPROVED** [review]

File: modules/ROOT/pages/7.1.1-release-notes.adoc (hunk 2 above)
Comment (kemister85) [reply modules/ROOT/pages/7.1.1-release-notes.adoc:68]:
```suggestion
==== A list with the start attribute, separated by `p` tags, resulted in incorrect numbering of lists.
```

Comment (kemister85) [reply modules/ROOT/pages/7.1.1-release-notes.adoc:68]: @tiny-ben-tran See sugested improvement.

**Pull request merged**